exp-player/addon/components/exp-card-sort/component.js

```diff
@@ -4,105 +4,105 @@ import layout from './template';
-    "goodImpression": "qsort.rsq.item.goodImpression",
-    "happeningOnce": "qsort.rsq.item.happeningOnce",
-    "honor": "qsort.rsq.item.honor",
-    "hostile": "qsort.rsq.item.hostile",
-    "humorous": "qsort.rsq.item.humorous",
-    "intellectuallyStimulating": "qsort.rsq.item.intellectuallyStimulating",
-    "intelligence": "qsort.rsq.item.intelligence",
-    "jobDone": "qsort.rsq.item.jobDone",
-    "masculinity": "qsort.rsq.item.masculinity",
-    "minorDetails": "qsort.rsq.item.minorDetails",
-    "money": "qsort.rsq.item.money",
-    "moralIssues": "qsort.rsq.item.moralIssues",
-    "music": "qsort.rsq.item.music",
-    "needsHelp": "qsort.rsq.item.needsHelp",
-    "negativeEmotions": "qsort.rsq.item.negativeEmotions",
-    "newRelationships": "qsort.rsq.item.newRelationships",
-    "noisy": "qsort.rsq.item.noisy",
-    "notClear": "qsort.rsq.item.notClear",
-    "oppositeSex": "qsort.rsq.item.oppositeSex",
-    "peopleGetAlong": "qsort.rsq.item.peopleGetAlong",
-    "physicalAttractiveness": "qsort.rsq.item.physicalAttractiveness",
-    "physicalThreats": "qsort.rsq.item.physicalThreats",
-    "physicallyActive": "qsort.rsq.item.physicallyActive",
-    "physicallyUncomfortable": "qsort.rsq.item.physicallyUncomfortable",
-    "playful": "qsort.rsq.item.playful",
-    "politics": "qsort.rsq.item.politics",
-    "positiveEmotions": "qsort.rsq.item.positiveEmotions",
-    "potentiallyEnjoy": "qsort.rsq.item.potentiallyEnjoy",
-    "power": "qsort.rsq.item.power",
-    "quickAction": "qsort.rsq.item.quickAction",
-    "rapidlyChanging": "qsort.rsq.item.rapidlyChanging",
-    "reassurance": "qsort.rsq.item.reassurance",
-    "reassuringPresent": "qsort.rsq.item.reassuringPresent",
-    "relevantHealth": "qsort.rsq.item.relevantHealth",
-    "religion": "qsort.rsq.item.religion",
-    "romanticPartners": "qsort.rsq.item.romanticPartners",
-    "ruminateDaydream": "qsort.rsq.item.ruminateDaydream",
-    "selfControl": "qsort.rsq.item.selfControl",
-    "sensations": "qsort.rsq.item.sensations",
-    "sexuality": "qsort.rsq.item.sexuality",
-    "shame": "qsort.rsq.item.shame",
-    "simpleClearcut": "qsort.rsq.item.simpleClearcut",
-    "smallAnnoyances": "qsort.rsq.item.smallAnnoyances",
-    "socialInteraction": "qsort.rsq.item.socialInteraction",
-    "successCooperation": "qsort.rsq.item.successCooperation",
-    "takenCareOf": "qsort.rsq.item.takenCareOf",
-    "talkingExpected": "qsort.rsq.item.talkingExpected",
-    "talkingPermitted": "qsort.rsq.item.talkingPermitted",
-    "tenseUpset": "qsort.rsq.item.tenseUpset",
-    "tryingImpress": "qsort.rsq.item.tryingImpress",
-    "underThreat": "qsort.rsq.item.underThreat",
-    "unhappySuffering": "qsort.rsq.item.unhappySuffering",
-    "unusualIdeas": "qsort.rsq.item.unusualIdeas",
-    "verbalFluency": "qsort.rsq.item.verbalFluency",
-    "workingHard": "qsort.rsq.item.workingHard",
-    "youFocus": "qsort.rsq.item.youFocus"
-};
+var cards = [
+    'qsort.rsq.item.potentiallyEnjoy',
```

Comment:
Just to verify: Did the actual name labels change in a way that would affect translations, or were all of these items simply re-ordered?